holidays/countries/india.py

```
@@ -492,12 +517,168 @@ def _populate_subdiv_br_public_holidays(self):
+
+        # Sant Guru Tekchand Maharaj Samadhi Utsav.
+        self._add_maharishi_valmiki_jayanti(tr("Sant Guru Tekchand Maharaj Samadhi Utsav"))
+
+        # Diwali (South India).
+        self._add_diwali_south_india(tr("Deepavali (South India)"))
+
+        # Dattatreya's Birthday.
+        self._add_dattatreya_jayanti(tr("Dattatreya's Jayanti"))
+
+    # Dadra and Nagar Haveli and Daman and Diu.
+    def _populate_subdiv_dh_public_holidays(self):
```

Ruff (0.16.0)

[warning] Missing return type annotation for private function `_populate_subdiv_dh_public_holidays`

Add return type annotation: `None`

(ANN202)